client/src/components/Register.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import {ThemeProvider, theme, CSSReset, Flex, Box, Heading, FormControl, FormLabel, Input, Button} from "@chakra-ui/react";
+
+const Register = () => {
```

Comment:
Register, like Login, should be a class. Create a function in the main Register component and pass it as props to the RegisterForm and call them when you submit the form.